site/2015/04/02/orm-2-5-0.rst

```diff
@@ -0,0 +1,1461 @@
+Doctrine ORM 2.5.0 Release
+==========================
+
+We are happy to announce the immediate availability of Doctrine ORM
+``2.5.0``.
+
+This release spans over almost 2 years of development, and is a major
+effort by the team and the community to make the ORM more robust and
+performing.
+
+`457
+issues <http://www.doctrine-project.org/jira/browse/DDC-3322?jql=project%20%3D%20DDC%20AND%20fixVersion%20%3D%202.5%20ORDER%20BY%20status%20DESC%2C%20priority%20DESC>`__
+were solved in this release, so we are very proud of the work being done
+by the community and the core team.
+
+What is new in 2.5.x?
+---------------------
+
+Doctrine ORM 2.5.0 comes with a set of major improvements:
+
+-  The Second Level Cache, a component that greatly improves ORM
```

Comment:
Should it be "Second-level Cache"? I assume capitalization is because it's a named feature, which is fine.